tests/compat.sh

```diff
@@ -534,6 +534,15 @@ add_mbedtls_ciphersuites()
     esac
 }
 
+# o_check_ciphersuite STANDARD_CIPHER_SUITE
```

Comment:
The input argument is a string with the format of "STANDARD CIPHER SUITE NAME = PROGRAM CIPHER SUITE NAME", how about changing it as?
```suggestion
# o_check_ciphersuite <"STANDARD_NAME=PROGRAM_NAME">
```